lib/Main.js

```diff
@@ -452,7 +453,7 @@ Main.prototype.onMatrixEvent = function(ev) {
     room.onMatrixMessage(ev).then(
         () => endTimer({outcome: "success"}),
         (e) => {
-            console.log("Failed: ", e);
+            log.error("Failed: ", e);
```

Comment:
While we're here, it might be worth reconsidering the message here. What failed?